### PR TITLE
Combine the 3 list types into a single lists documentation page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Combine the 3 list types into a single lists documentation page

## QA
- Check the code that no content is lost

## Details
Fixes https://github.com/ubuntudesign/docs.vanillaframework.io/issues/101

### CODE REVIEW

File: docs/en/patterns/lists.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+collection: patterns
+title: Lists
+---
+
+# Lists
+If you want to display lists in a way that is more visually distinctive than
+the standard <ol> and <ul>, Vanilla brochure theme has 3 list styles at your
```
Comment (barrymcgee) [inline docs/en/patterns/lists.md]: Do these html tags need backticks?

File: docs/en/patterns/lists.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+disposal.
+
+
+## Ticked List
```
Comment (barrymcgee) [inline docs/en/patterns/lists.md]: Should list be capitalised?

**Comment by barrymcgee** [comment]
@anthonydillon Just some queries in code.

**Comment by anthonydillon** [comment]
@barrymcgee done and ready for your eyes again.

**Pull request merged**